<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ValueListInputDialog</class>
 <widget class="QWidget" name="ValueListInputDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>221</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>List</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="m_toolbarLayout">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QToolButton" name="m_addToolButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../Resources/Resources.qrc">
         <normaloff>:/icons/add.png</normaloff>:/icons/add.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="m_loadToolButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../Resources/Resources.qrc">
         <normaloff>:/icons/folder_out.png</normaloff>:/icons/folder_out.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="m_deleteToolButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../Resources/Resources.qrc">
         <normaloff>:/icons/delete_small.png</normaloff>:/icons/delete_small.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="m_toolbarSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="m_valueListWidget">
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustIgnored</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="columnCount">
      <number>1</number>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <column/>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="m_buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../Resources/Resources.qrc"/>
 </resources>
 <connections/>
</ui>
